### PR TITLE
Add startTime to player docs

### PR DESCRIPTION
**Pull request by third774**
cc @zaidf

### CODE REVIEW

File: products/stream/src/content/viewing-videos/using-the-stream-player.md [hunk 1]
```diff
@@ -111,4 +111,8 @@ Player options are configured with querystring parameters in the iframe's `src`
 
   - The video id from the video you've uploaded to Cloudflare Stream should be included here.
 
+- `startTime`
+
+  - A timestamp that the video will start at. If a plain number is used such as `?startTime=123`, it will be interepreted as `123` seconds. More human readable timestamps can also be used, such as `?startTime=1h12m27s` for `1 hour, 12 minutes, and 27 seconds`.
```
Comment (dcpena) [inline products/stream/src/content/viewing-videos/using-the-stream-player.md]: What about "A timestamp that specifies when the video begins." ?

**Review by dcpena — APPROVED** [review]
Approving because my comment is a suggestion and non-blocking

File: products/stream/src/content/viewing-videos/using-the-stream-player.md (hunk 1 above)
Comment (third774) [reply products/stream/src/content/viewing-videos/using-the-stream-player.md]: Iterating based on that, how do you feel about "A timestamp that specifies the time from which playback will begin."?

I like that it kinda distinguishes between the video itself and playback.

Comment (dcpena) [reply products/stream/src/content/viewing-videos/using-the-stream-player.md]: Ah, good callout! Can we iterate one more time? haha.

"A timestamp that specifies the time when playback begins."

Comment (third774) [reply products/stream/src/content/viewing-videos/using-the-stream-player.md]: Adopted in a force push!

**Pull request merged**